__tests__/news.test.js

```diff
@@ -48,3 +49,35 @@ describe("/api/topics", () => {
     });
   });
 });
+
+//Test 3 - Get articles by ID
+describe("/api/articles/:article_id", () => {
+  describe("GET Article object by specific article ID", () => {
+    test("200: responds with an Object of a specific article", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body;
```

Comment:
from your controller it looks like you are sending back the data on body.article.  double check that you are running your tests before commiting.